README.md

```diff
@@ -87,22 +87,23 @@ python merge_manifests.py --output-path merged_manifest.csv --merge-dir all-mani
 
 ### Training a Model
 
-Configuration is done via [Hydra](https://github.com/facebookresearch/hydra), and the config file can be seen [here](./config.yaml). 
-You can either edit the defaults in the config file, or override via the cmdline as done below.
+Configuration is done via [Hydra](https://github.com/facebookresearch/hydra).
+
+Defaults can be seen in [config.py](./deepspeech_pytorch/config.py). Below is how you can override values set already:
 
 ```
 python train.py data.train_manifest=data/train_manifest.csv data.val_manifest=data/val_manifest.csv
 ```
 
-Use `python train.py --help` for more parameters and options.
+Use `python train.py --help` for all parameters and options.
```

Comment:
By the way, you can customize your help a bit:
https://hydra.cc/docs/configure_hydra/app_help

Check the examples in Hydra for a complete example (I need to update the doc to link to it).

Reply:
Awesome! any chance helper message per argument is on the roadmap? I guess it's not the cleanest thing to implement though

Reply:
I am going to try something better eventually.

https://github.com/omry/omegaconf/issues/131